Header header

uint32 gear

float64 priority
float64 lifetime